services/businessconfig/src/main/modeling/swagger.yaml

```diff
@@ -537,6 +537,8 @@ definitions:
         type: boolean
       lttdVisible: 
         type: boolean
+      recipientVisible: 
+        type: boolean
```

Comment:
It could be interesting to have a short description here like for other properties, so people wanting to send cards see at once what this property impacts without having to go look in the adoc documentation.
From the name alone it could mean displaying the recipients in the card rather than displaying the recipients dropdown.